website/templates/emails/user_message_institutional_access_request.html.mako

```diff
@@ -0,0 +1,34 @@
+<%inherit file="notify_base.mako" />
+
+<%def name="content()">
+    <tr>
+      <td style="border-collapse: collapse;">
+        <%!from website import settings%>
+        Hello ${recipient.fullname},
+        <p>
+            ${sender.fullname} from your <b>${institution.name}</b>, has sent you a request regarding your project.
+        </p>
+        % if message_text:
+        <p>
+            Message from ${sender.fullname}:<br>
+            ${message_text}
+        </p>
+        % endif
+        <p>
+            To review this request, please visit your project dashboard or contact your institution administrator for further details.
+        </p>
```

Comment:
This isn't accurate for this kind of email. Please check with product to verify what the text of the email surrounding the comment will be when it's not a request for access message. Also, the name of the email template is not correct for this feature, since it's not a request for access message.

Reply:
Sorry about that I simply copied the wrong message Product gave me, here's their docs if you want to confirm scroll down to the end. https://docs.google.com/document/d/1ZVbfohipKiJrMeRQIp3xCIdO_l9kyMVPui1-CP0teBs/edit?tab=t.0